cli/command/formatter/node.go

```diff
@@ -0,0 +1,104 @@
+package formatter
+
+import (
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/cli/command"
+)
+
+const (
+	defaultNodeTableFormat = "table {{.ID}} {{.Self}}\t{{.Hostname}}\t{{.Status}}\t{{.Availability}}\t{{.ManagerStatus}}"
+
+	nodeIDHeader        = "ID"
+	selfHeader          = " "
```

Comment:
Thanks @AkihiroSuda for the review. The PR has been updated. Please take a look.